From d433290bfb9583619448ca94195fa607412807d7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Fri, 15 Jan 2021 08:31:11 +0100
Subject: [PATCH] do not install a global 'tests' package

setuptools.find_packages() finds all valid packages by default that
incidentally includes the test directory that gets installed as global
'tests'.  Add an explicit exclude to avoid that.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 791850e..ad39260 100644
--- a/setup.py
+++ b/setup.py
@@ -28,7 +28,7 @@ if __name__ == '__main__':
         description=about['__description__'],
         url=about['__url__'],
         license='BSD',
-        packages=setuptools.find_packages(),
+        packages=setuptools.find_packages(exclude=['tests']),
         long_description=long_description,
         install_requires=['six'],
         tests_require=['pytest'],
-- 
2.30.0

